content/docs/guides/partner-billing.md

```diff
@@ -0,0 +1,346 @@
+---
+title: Manage billing with consumption limits
+subtitle: Learn how to set usage quotas per project with the Neon API
+enableTableOfContents: true
+isDraft: false
+updatedOn: '2023-11-03T15:15:02.548Z'
+---
+
+When setting up your billing solution with Neon, you may want to impose some hard limits on how much storage or compute size a given project can consume. For example, you may want to cap how much usage your free tier users can consume versus pro or enterprise users. With the Neon API, you can use the `quota` key to set usage limits for a variety of consumption metrics. These limits act as thresholds after which all active computes for a project are [suspended](#what-happens-when-the-quota-is-met). 
+
+## Metrics and quotas
+By default, Neon tracks a variety of consumption metrics at the project level. If you want to set quotas (max limits) for these metrics, you need to explicitly [configure](#configuring-quotas) them. 
+
+### Available metrics
+
+Here are the relevant metrics that you can set project-level quotas for:
+
+* `active_time_seconds`
+* `compute_time_seconds`
+* `written_data_bytes`
+* `data_transfer_bytes` 
+
+These consumption metrics represent total cumulated usage across all branches and computes in a given project, accrued so far in a given monthly billing period. They are refreshed on a set day every month, on whichever date your new billing period starts. 
```

Comment:
Is it any date or is it the first day of the month that a new billing period starts? We'll have to check. I always thought it was the first of the month, but I could be wrong.

Reply:
Revised to: 

"Metrics are refreshed on the first day of the following month."